app/services/data_migrations/incorrect_equality_and_diversity_migration.rb

```diff
@@ -0,0 +1,56 @@
+module DataMigrations
+  class IncorrectEqualityAndDiversityMigration
+    TIMESTAMP = 20240225202828
+    MANUAL_RUN = true
+    OLD_HESA_VALUES = <<-SQL.freeze
+      (equality_and_diversity->'hesa_sex' ?| array['1', '2', '3']) OR
+      (equality_and_diversity->'hesa_ethnicity' ?| array['10','15','21','22','29','31','32','33','34','39','41','42','43','49','50','80','90','98']) OR
+      (equality_and_diversity->'hesa_disabilities' ?| array['00','08']) OR
+      (equality_and_diversity->'disabilities' ?| array['Learning difficulty', 'Social or communication impairment', 'Long-standing illness', 'Physical disability or mobility issue', 'Deaf', 'Blind'])
+    SQL
+
+    def change
+      records.find_each do |application_form|
+        hesa_converter = HesaConverter.new(
+          application_form:,
+          recruitment_cycle_year: RecruitmentCycle.current_year,
+        )
+
+        application_form.update_columns(
```

Comment:
Audit comment is not an attribute available at update_columns 🤔. I am avoiding trying to call update or save because I don't want to trigger any callbacks. I can see on audit gem people has the same sort of issue.